src/index.js

```diff
@@ -1,60 +1,58 @@
 import './style.css';
-import getTasks, { addTask } from './functions.js';
 import statueUdpdateLS, { statueUdpdateUI } from './interactive.js';
+import addItemUI, { diplayTask, clear } from './uImanpulate.js';
+import Task from './task.js';
+import getDataLS, { addTaskLS, indexValue, remove } from './localstorage.js';
+import editText from './add-remove.js';
 
-const tasks = [
-  { description: 'wash the dashes', completed: false, index: 0 },
-  { description: 'complete to do ', completed: false, index: 1 },
-  { description: 'wash the dashes', completed: false, index: 2 },
-];
```

Comment:
The data should be take from the user and not hard coded.

Reply:
It is already deleted code

Reply:
Thank you